MeesGame/Gameplay/Level/Level.cs

```diff
@@ -58,7 +59,9 @@ public void ReadXml(XmlReader reader)
 
         public void WriteXml(XmlWriter writer)
```

Comment:
This class is never serialized I believe, so it does not need to implement IXmlSerializable